aot-tests/src/test/resources/com/dylibso/chicory/testing/MethodTooLargeTest.testBigFunc.approved.txt

```diff
@@ -0,0 +1,752 @@
+public final class com/dylibso/chicory/$gen/CompiledMachine implements com/dylibso/chicory/runtime/Machine {
```

Comment:
This output is too big to be effectively checked with an approval.

With "golden master" testing we should be very careful to only add things that brings value as they are usually very noisy.

Reply:
Yeah.. the main thing I was interested in was making sure the big function was not getting AoT.  Let me see if I can narrow down the output to just that function.

Reply:
narrowed it down.